debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -419,12 +504,23 @@ public void validate() {
         if (!Strings.isNullOrEmpty(columnExcludeList) && !Strings.isNullOrEmpty(columnIncludeList)) {
             throw new ConnectException("Cannot define both column.exclude.list and column.include.list. Please specify only one.");
         }
+
+        validateColumnNamingProperties();
```

Comment:
Could we use the `Field#withValidation` handler for this rather than calling this directly?